lib/pure/htmlgen.nim

```diff
@@ -8,11 +8,11 @@
 #
 
 ## Do yourself a favor and import the module
-## as `from htmlgen import nil` and then fully qualify the macros.
+## as `from std/htmlgen import nil` and then fully qualify the macros.
 ##
-## *Note*: The Karax project (`nimble install karax`) has a better
-## way to achieve the same, see https://github.com/pragmagic/karax/blob/master/tests/nativehtmlgen.nim
-## for an example.
+## .. Note:: The Karax project (`nimble install karax`) has a better
```

Comment:
btw there are other notations too. For example `.. Tips::` or so.

> admonitions: "attention", "caution", "danger", "error", "hint", "important", "note", "tip", "warning", "admonition"
https://nim-lang.github.io/Nim/rst.html

Reply:
![temp](https://user-images.githubusercontent.com/1189414/110234959-1430a700-7f0c-11eb-9cc2-9719a3d81ec0.png)

Reply:
I see

Reply:
I see